src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/PostgresLedgerExtenderService.cs

```diff
@@ -1451,17 +1451,50 @@ void AggregateEntityResourceVaultInternal(long entityId, long resourceEntityId,
                     _ => throw new ArgumentOutOfRangeException(nameof(x), x, null),
                 };
 
+                var isDeleted = balance.IsZero();
```

Comment:
How accurate the withdrawals / vault balances are? @dhedey ? I'm asking cause I wonder if it is reasonable to expect `0.000000000000001` - similarly to how you have to handle such cases with floating point.

Reply:
it's no longer relevant as I've removed that piece of code.